tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -1699,13 +1721,23 @@ mod follow_impls {
 
     #[test]
     fn to_struct() {
+        use flatbuffers::EndianScalar;
         #[derive(Copy, Clone, Debug, PartialEq)]
         #[repr(C, packed)]
         struct FooStruct {
```

Comment:
Please add a comment describing that we're acting as if we are the code generator by making `FooStruct`. @tymcauley If you think it would be better to use the real generated `Vec3` for this, too, I'm fine with that.

Reply:
That sounds like a good idea. Also, any reason not to make `FooStruct` common to the whole `follow_impls` module, since we now have 3 identical implementations?

Reply:
See 828db84 for possible resolution.

Reply:
@tymcauley perfect!